chainer/functions/connection/deformable_convolution_2d_sampler.py

```diff
@@ -0,0 +1,143 @@
+import numpy
+
+from chainer import cuda
+
+from chainer.functions.array.broadcast import broadcast_to
+from chainer.functions.array.concat import concat
+from chainer.functions.array.pad import pad as pad_func
+from chainer.functions.array.spatial_transformer_sampler import\
+    spatial_transformer_sampler
+from chainer.functions.math.matmul import matmul
```

Comment:
Could you replace these 6 lines of 5 function imports with module-level imports? Importing functions directly from other modules is not recommended according to OpenStack style guidelines.

Reply:
@niboshi Do you know the background of this convention?

Reply:
@yuyu2172 I found this comment. https://github.com/chainer/chainer/issues/26#issuecomment-113058854
> Following offline discussions, we will use PEP8 with OpenStack Style Guidelines as our coding style.
So basically we need to follow the OpenStack Style Guidelines.